editor/src/clj/editor/display_profiles.clj

```diff
@@ -123,13 +131,18 @@
 
 (g/defnode DisplayProfilesNode
   (inherits resource-node/ResourceNode)
+  (property auto-layout-selection g/Bool (default true)
+            (dynamic visible (g/constantly false)))
```

Comment:
When I started, I was sure that there should be a way to avoid property creation and operate directly with protobuf. But I didn't figure out how to do so. Please let me know if it can be simlified.